index.js

```diff
@@ -48,7 +53,7 @@ Merry.prototype.defaultRoute = function (handler) {
 
   function routeHandler (req, res, params) {
     var ctx = new Ctx(req, res, self.log)
-    ctx.params = params
+    ctx.params = params.params
```

Comment:
[this is the object](https://github.com/yoshuawuyts/server-router/blob/master/index.js#L24) that server-router returns, so I have to do `params.params` to actually return the given params. So I am not quite sure what you mean.